core/src/com/mygdx/game/models/Round.java

```diff
@@ -0,0 +1,114 @@
+package com.mygdx.game.models;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+/**
+ * The round keeps track of current players and their
+ * BrainstormingPhase, EliminationPhase, Brains being used the round,
+ * and which brain the player is currently on.
```

Comment:
Minor grammar complaint. Not really sure what this sentence is supposed to mean

Reply:
The round keeps track of which players are in the round, and also the brainstormingphase, eliminationphase, brains and the current brain nr assosieted with the players. I can try and write it more informative.

Reply:
How about: "The round keeps track of which players are in the round, and also the brainstormingphase, eliminationphase, brains and the current brain nr. for each player."?